README.md

```diff
@@ -165,6 +223,50 @@ would become the following in beancount:
     2018-02-02 * "Train Brussels airport to city" ^2018-02-02-brussels-fosdem #debian
 
 
+### Comments
+
+Comments are supported.  Since beancount currently [doesn't allow
+full-line comments within
+transactions](https://bitbucket.org/blais/beancount/issues/143), these
+are stored as metadata.
+
+
+### Lots
+
+Lot costs and prices are supported, including per-unit and total lot
+costs.  Lot dates are converted to beancount but lot notes are not
+handled at the moment.
+
+The behaviour of ledger and beancount is different when it comes to
+costs.  In ledger, the statement
+
+    Assets:Test          10.00 EUR @ 0.90 GBP
+
+creates the lot `10.00 EUR {0.90 GBP}`.  In beancount, this is not the
+case and a cost is only associated if done so explicitly:
+
+    Assets:Test          10.00 EUR {0.90 GBP}
+
+This makes automatic conversion tricky because some statements should be
+simple conversions without associating a cost whereas it's vital to
+preserve the cost in other conversions.
+
+Generally, it doesn't make sense to preserve the cost for currency
+conversion (as opposed to conversions involving commodities like shares
+and stocks).  Since most currency tickers consist of 3 characters (EUR,
```

Comment:
I don't know what a "currency ticker" is, but maybe it's just me :-)

Reply:
I think "currency code" is a better term.